backend/application/services/queue/consumer.py

```diff
@@ -1,90 +1,49 @@
-from asyncio import sleep
```

Comment:
The resume of the PR:
```
async def consumer():
    ...
        while True:
            queued_tasks: Sequence[Message] = queue_tasks.receive()
            coroutines = []
            for queued_task in queued_tasks:
                ...
                if task_record.name in TASK_MAP:
                    if task_kind == MessageKind.task:
                        repository.mark_running(obj=task_record)
                        coroutine = await TASK_MAP[task_record.name](task_id=task_id)
                        coroutines.append(coroutine)
                 ...
            if coroutines:
                results = await gather(*coroutines)
                logging.info(f"all results {results}")
                for task_result, queued_task in zip(results, queued_tasks):
                    if task_result == -1:
                        repository.mark_failed(pk=queued_task.body)
                    else:
                        repository.mark_done(pk=queued_task.body)
```

while on _TASK_MAP[task_record.name](task_id=task_id)_


```
async def with_timeout(coroutine, timeout, **kwargs):
    try:
        result = await asyncio.wait_for(coroutine(**kwargs), timeout=timeout)
        return result
    except asyncio.TimeoutError:
        return -1  # defined timeout as -1


def task(task_name: str):
    def wrapper(func):
        async def async_wrapper(**kwargs):
            return with_timeout(coroutine=func, timeout=MAX_TIMEOUT_SECONDS, **kwargs)
        return async_wrapper
    return wrapper
```